templates/frontend/objects/article_details.tpl

```diff
@@ -215,10 +215,10 @@
 				<div class="item cover_image">
 					<div class="sub_item">
 						{if $article->getLocalizedCoverImage()}
-							<img src="{$article->getLocalizedCoverImageUrl()|escape}"{if $article->getLocalizedCoverImageAltText()} alt="{$article->getLocalizedCoverImageAltText()|escape}"{/if}>
+							<img src="{$article->getLocalizedCoverImageUrl()|escape}"{if $article->getLocalizedCoverImageAltText()} alt="{$article->getLocalizedCoverImageAltText()|default:'null'}"{/if}>
```

Comment:
Because the `{if}` tag is used here, when alt text is missing the whole `alt` attribute will be missing. So when you find things like this, you can use:

```
<img src="{$article->getLocalizedCoverImageUrl()|escape}" alt="{$article->getLocalizedCoverImageAltText()|default:'null'}">
```

Reply:
Same for several examples below.